libs/common/src/types/mongo-db-models/activity-author.ts

```diff
@@ -0,0 +1,5 @@
+export type ActivityAuthor = {
```

Comment:
I dont think this necessarily should be in mongo-db-models. It also might be used in more places than just activities but happy to leave as `ActivityAuthor` for now